### ISSUE TITLE
I cannot select a specific commitment

### ISSUE THREAD

**Issue by seriousben** [QlikHackathon/decode-oct-2017#7]
Clicking on a commitment brings me to its page. We need a way to select that is separate from clicking to go to the page.

The solution needs to be the same for all lists.

**Comment by rjriel** [comment]
wondering what your thought is behind this. I had always considered the commitment to be the finite data people are eventually trying to drill down to, so I'm wondering why it would need to be selectable.

**Comment by Tuanhao** [comment]
Hey, I have an idea for making the commitment more accessible. I also implemented it, but I don't know how to create a new branch and a new PR. It must be something with the permission.

**Comment by seriousben** [comment]
@Tuanhao I am on it.

**Comment by seriousben** [comment]
@Tuanhao This should be fixed now, please let me know.

**Comment by Tuanhao** [comment]
So PR #10 will have a panel on a side of the commitments list. It is just a prototype, no styling yet. What do you guys think?

**Comment by rjriel** [comment]
I like the idea, although styling definitely needs to be done. The description information can definitely be done better though, having a fetch go out each time you click can not only have issues with performance (network, slicing through the content, etc), but with the engine available to us and the fact that those descriptions are more than likely to remain static, I think it would be much better to either ask the UN if they have this data available or do a one time scrape of this, then have the engine ingest that data into the app.

**Comment by rjriel** [comment]
further to that last comment, if we were to ingest the descriptions into the app we may find a use for integrating the search capabilities built of the engine into our app

**Comment by Tuanhao** [comment]
True. It would be better if we make a one time scraping thing. Or we can show this to the UN (if possible) and they may provide us the information, then we can add it to the engine. I dont know scrapping and much about engine too, so it will be a great experience for me to learn.
The purpose of my PR is to solve this issue, if it does solve the problem, then i will start working on the styling part.
And i also have no clue about the search capabilities both purpose and implementation!
